app/src/main/java/org/projectbuendia/client/providers/Contracts.java

```diff
@@ -36,7 +36,8 @@
         ORDERS("orders"),
         PATIENTS("patients"),
         USERS("users"),
-        SYNC_TOKENS("sync_tokens");
+        SYNC_TOKENS("sync_tokens"),
+        UNSYNC_TOKENS("unsync_forms");
```

Comment:
What's the UX story for this?  Sorry if I've missed a design discussion
about this.

As I last remember, we weren't going to do offline submission because there
were too many unsolved UX design issues related to users understanding when
their changes had actually been saved, and how to handle edit conflicts.
Since we thought that not knowing which orders had been administered could
lead to life-threatening mistakes, we had decided it was too risky.

Were these design issues ever addressed?  If they weren't, I would be very
afraid to land this feature.
On Jan 7, 2016 1:56 PM, "Fabian Tamp" <notifications@github.com> wrote:

> In app/src/main/java/org/projectbuendia/client/providers/Contracts.java
> <https://github.com/projectbuendia/client/pull/172#discussion_r49043156>:
>
> > @@ -36,7 +36,8 @@
> >          ORDERS("orders"),
> >          PATIENTS("patients"),
> >          USERS("users"),
> > -        SYNC_TOKENS("sync_tokens");
> > +        SYNC_TOKENS("sync_tokens"),
> > +        UNSYNC_TOKENS("unsync_forms");
>
> This should be consistent with the data type you've created and the table
> name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projectbuendia/client/pull/172/files#r49043156>.
>